src/schema/uuid/alias/utils.ts

```diff
@@ -39,10 +44,21 @@ export function encodePath(path: string) {
 }
 
 export function createAliasResolvers<
-  T extends { alias: string | null }
+  T extends { alias: string | null } & AbstractUuidPayload
```

Comment:
```suggestion
  T extends { alias: string | null } & AbstractUuidPayload & { instance?: Instance }
```

Doesn't this work in the meanwhile?